ivadomed/utils.py

```diff
@@ -433,8 +433,9 @@ def segment_volume(folder_model, fname_image, fname_roi=None):
     preds_list, slice_idx_list = [], []
     for i_batch, batch in enumerate(data_loader):
         with torch.no_grad():
-            img = batch['input']
+            img = batch['input'].cuda() if torch.cuda.is_available() else batch['input']
```

Comment:
What about using:
https://github.com/neuropoly/ivadomed/blob/master/ivadomed/utils.py#L469

Reply:
Changed it!